tests/onnxruntime/test_modeling.py

```diff
@@ -1452,6 +1452,13 @@ def test_compare_generation_to_io_binding(self, model_arch):
 class ORTModelForSpeechSeq2SeqIntegrationTest(unittest.TestCase):
     SUPPORTED_ARCHITECTURES = ("whisper",)
 
+    def _generate_random_audio_data(self):
```

Comment:
Can this be useful for the audio input generator?

Reply:
This generates an audio signal however `DummyAudioGenerator` requires to generate preprocessed audio data which are Mel Spectrograms. We could add the entire process of preprocessing audio data in `DummyAudioGenerator` (similar to how feature_extractor is implemented), but it may be not be too much  useful for exporting the model.